docs/section19/19-14.md

```diff
@@ -0,0 +1,71 @@
+                - Check if the total relief fan power is greater than 0 in the proposed, if yes then set is_modeled_with_relief_fan_in_proposed to true: `if modeled_relief_fan_power_p > 0: is_modeled_with_relief_fan_in_proposed = true`  
+                            
+
+            - Create fan system object: `fan_sys_b = hvac.fan_system`  
+            - Get list from dictionary of the hvac system's fan system's supply, return, exhaust and relief fan CFM: `fan_sys_CFM = list(get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM(B_RMI,fan_sys_b)["Fan_Power"]["Fan_CFM"])`  
+            - Get the hvac system total supply fan cfm, need to get the first element in the list of fan cfms (this assumes that all fans associated with the supply_fans object are in parallel (i.e., if multiple fans the cfm is additive.):`hvac_sys_total_supply_fan_cfm_b = fan_sys_CFM[0]` 
+            - Calculate 90% of the supply air quantity: `90_percent_supply_cfm = 90% * hvac_sys_total_supply_fan_cfm_b`   
+            - Calculate the supply CFM minus the minimum OA CFM for the fan system: `supply_minus_OA_cfm = hvac_sys_total_supply_fan_cfm_b - fan_sys_b.minimum_outdoor_airflow`  
+            - Check if is_modeled_with_return_fan_in_proposed == true, if so then set the modeled_cfm equal to the return cfm (second element in list of baseline hvac fan cfms): `if is_modeled_with_return_fan_in_proposed == true: modeled_cfm = fan_sys_CFM[1]`  
+            - Else, is_modeled_with_relief_fan_in_proposed == true, then set the modeled_cfm equal to the relief cfm (fourth element in list of baseline hvac fan cfms): `Else: modeled_cfm = fan_sys_CFM[3]`   
+
+            - **Rule Assertion:** 
+            - Case 1: If return or relief fans are specified in the proposed design (but not both) and the baseline building design is modeled with fans serving the same functions and sized for the baseline system supply fan air quantity less the minimum outdoor air, or 90% of the supply fan air quantity, whichever is larger then pass: `if (all(is_modeled_with_return_fan_in_proposed, is_modeled_with_relief_fan_in_proposed) == false) and (is_modeled_with_return_fan_in_proposed == true or is_modeled_with_relief_fan_in_proposed == true) and (modeled_cfm == max(supply_minus_OA_cfm,90_percent_supply_cfm)): outcome = "PASS"`  
+            - Case 2: Else if both return or relief fans are specified in the proposed design then outcome is UNDETERMINED: `elif all(is_modeled_with_return_fan_in_proposed, is_modeled_with_relief_fan_in_proposed) == true: outome = "UNDETERMINED" and raise_message "Return and relief fans both appear to be modeled in the proposed design, conduct manual check that the baseline building design is modeled with fans serving the same functions and sized for the baseline system supply fan air quantity less the minimum outdoor air, or 90% of the supply fan air quantity, whichever is larger per Section G3.1.2.8.1."`  
```

Comment:
In this case, should we check that baseline has both return and relief fans and that their total airflow meets the requirement?

Reply:
Adjust depending on above discussions.

Reply:
For the purpose of this RDS, whenever fans are included in the proposed should also be included in the baseline. 

Total cfm (return + relief) is the 90%. Check for total and not distribution. 

Exhaust cfm same as in proposed (ask Mike's about this).

Reply:
Updated per everything except exhaust because Mike R agreed. This rule does not discuss exhaust, shall I add another rule? If so, not sure which section of App G to reference since its not in the Standard?